Packs/CommonPlaybooks/Playbooks/playbook-IP_Enrichment_-_Internal_-_Generic_v2.yml

```diff
@@ -470,24 +554,40 @@ inputs:
   playbookInputQuery:
 - key: InternalRange
   value:
-    simple: 172.16.0.0/12,10.0.0.0/8,192.168.0.0/16
+    simple: inputs.InternalRange
   required: false
   description: "A CSV list of IP address ranges (in CIDR notation). Use this list to check if an IP address is found within a set of IP address ranges. \nFor example: \"172.16.0.0/12,10.0.0.0/8,192.168.0.0/16\" (without quotes). If no list is provided, will use default list provided in the IsIPInRanges script (the known IPv4 private address ranges)."
   playbookInputQuery:
 - key: ResolveIP
   value:
-    simple: "True"
+    complex:
+      root: inputs.ResolveIP
   required: true
   description: |-
     Whether to convert the IP address to a hostname using a DNS query (True/False).
     The default value is true.
   playbookInputQuery:
+- key: ExecutedFromParent
+  value:
+    simple: "False"
+  required: false
+  description: |-
+    Whether to execute common logic, like the classification of IP addresses to ranges and resolving, in the the main (IP Enrichment - Generic v2) enrichment playbook, instead of in the sub-playbooks.
```

Comment:
```suggestion
    Whether to execute common logic, like the classification of IP addresses to ranges and resolving, in the the main (IP Enrichment - Generic v2) enrichment playbook, instead of in the sub-playbooks.
    
    Possible values are: True, False.
```